doc/python/px-arguments.md

```diff
@@ -239,7 +233,7 @@ fig.show()
 
 ### Passing dictionaries or array-likes as the data_frame argument
 
-The column-based argument `data_frame` can also be passed with a `dict` or `array`. Using a dictionary can be a convenient way to pass column names used in axis titles, legend entries and hovers without creating a pandas DataFrame.
+The column-based argument `data_frame` can also be passed with a `dict` or `array`. Using a dictionary can be a convenient way to pass column names used in axis titles, legend entries and hovers without creating a DataFrame.
```

Comment:
why "data_frame" in this case?

Reply:
In this case it's name of the px parameter.